### PR TITLE
db backup restore threat

### PR DESCRIPTION
**Pull request by ianwalkersmithciticom**
It is possible for a threat actor to write backup over a database, thereby overwriting the db with restored copy.

### CODE REVIEW

**Comment by eddie-knight** [comment]
@sshiells-scottlogic @dogle-scottlogic @damienjburks — Who can bring the link checker into a good state? Looks like it needs to be disabled or a standalone PR created to resolve the findings

**Comment by damienjburks** [comment]
@eddie-knight i got you. Working on it now. Will be resolved in the next 30

**Comment by damienjburks** [comment]
@sshiells-scottlogic  was able to update it before me. We should be good to move forward @eddie-knight

**Comment by sshiells-scottlogic** [comment]
@eddie-knight  we pushed  a quick fix for now that should unblock this. 

Will look for a proper fix asap

**Comment by sshiells-scottlogic** [comment]
@damienjburks  @eddie-knight  a better fix is available https://github.com/finos/common-cloud-controls/pull/568

**Comment by damienjburks** [comment]
@ianwalkersmithciticom please fix your yaml-lint checks.

File: services/database/relational/threats.yaml
Comment (eddie-knight) [inline services/database/relational/threats.yaml:105]:
```suggestion
    description: threat actor uses brute force attack to discover
    database user password, threat actor then has access to the
    database user
```

**Comment by eddie-knight** [comment]
Checks are good, now just needs approval from @finos/ccc-wg-security

**Review by mlysaght2017 — APPROVED** [review]

**Pull request merged**
